### ISSUE TITLE
device_tracker: not marked as made by "Virtual" integration

### ISSUE THREAD

**Issue by ildar170975** [twrecked/hass-virtual#65]
There some `device_tracker` entities:
```
device_tracker:
  - platform: virtual
    devices:
      - name: virtual_tracker_1
        peristent: True
        location: home

      - name: virtual_tracker_2
        peristent: False
        location: not_home

      - name: virtual_tracker_3
        peristent: False
        location: not_home
```
And these entities are not marked as created by the "Virtual" integration:

![изображение](https://github.com/twrecked/hass-virtual/assets/71872483/6b063d05-0fd3-402f-8794-504581300cbd)

For most other platforms an integration is displayed properly:

![изображение](https://github.com/twrecked/hass-virtual/assets/71872483/30b4cecb-646f-4cfa-93d3-1266ebac64e6)

**Comment by twrecked** [comment]
Give me a week or two. I'm moving my other components to config flow and then I'll do this one

**Comment by ildar170975** [comment]
I think the issue can be closed since in 0.9 all trackers are marked as belonging to "Virtual Devices":
![image](https://github.com/user-attachments/assets/6069d714-6b6c-438c-aaa5-1d80e998201f)
